tools/wake-unit/cli_options.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2023 SiFive, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You should have received a copy of LICENSE.Apache2 along with
+ * this software. If not, you may obtain a copy at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "../tools/wake/cli_options.h"
+
+#include <string.h>
+
+#include "unit.h"
+
+TEST(cli_options_basic) {
+  {
+    int argc = 2;
+    char *argv[] = {(char *)"wake", (char *)"--help", nullptr};
```

Comment:
When you construct an array like this you can use `sizeof(argv)/sizeof(*argv)` to get the number of elements. That's better than just setting it to 2.